src/Listener/ApiListener.php

```diff
@@ -151,7 +152,8 @@ protected function _checkRequestMethods()
      */
     public function registerExceptionHandler()
     {
-        Configure::write('Error.exceptionRenderer', 'Crud\Error\ExceptionRenderer');
+        $config = ['exceptionRenderer' => 'Crud\Error\ExceptionRenderer'];
```

Comment:
Then it shouldn't be called `registerExceptionHandler()`, and the registration should be left to the user. It just mixes up to see it there, no mention in the docs and then have to dig to figure it out.

In any case, it wasn't working, even when manually registered in bootstrap for validation errors because of the error with `ExceptionRenderer::validation()`.